### PR TITLE
J22 10 Cards 

### PR DESCRIPTION
**Pull request by Simisays**
All cards are working during tests

### CODE REVIEW

**Comment by Hanmac** [comment]
> Whenever you cast a creature spell with mana value 5 or greater, that creature enters the battlefield with X additional +1/+1 counters on it, where X is its mana value minus 4.

```
T:Mode$ SpellCast | ValidCard$ Creature.cmcGE5 | ValidActivatingPlayer$ You | Execute$ TrigEffect | TriggerZones$ Battlefield | TriggerDescription$ Whenever you cast a creature spell with mana value 5 or greater, that creature enters the battlefield with X additional +1/+1 counters on it, where X is its mana value minus 4.
SVar:TrigEffect:DB$ Effect | RememberObjects$ TriggeredCard | ForgetOnMoved$ Stack | ReplacementEffects$ ETBCreat
SVar:ETBCreat:Event$ Moved | ValidCard$ Card.IsRemembered | Destination$ Battlefield | ReplaceWith$ DBPutP1P1 | ReplacementResult$ Updated | Description$ That creature enters the battlefield with X additional +1/+1 counters on it, where X is its mana value minus 4.
SVar:DBPutP1P1:DB$ PutCounter | Defined$ ReplacedCard | CounterType$ P1P1 | ETB$ True | CounterNum$ Y
```

the only thing missing is to calculate Y, but for this i'm not sure if the value is calculated when the Trigger resolves or when the Creature enters

**Comment by Simisays** [comment]
> > Whenever you cast a creature spell with mana value 5 or greater, that creature enters the battlefield with X additional +1/+1 counters on it, where X is its mana value minus 4.
> 
> ```
> T:Mode$ SpellCast | ValidCard$ Creature.cmcGE5 | ValidActivatingPlayer$ You | Execute$ TrigEffect | TriggerZones$ Battlefield | TriggerDescription$ Whenever you cast a creature spell with mana value 5 or greater, that creature enters the battlefield with X additional +1/+1 counters on it, where X is its mana value minus 4.
> SVar:TrigEffect:DB$ Effect | RememberObjects$ TriggeredCard | ForgetOnMoved$ Stack | ReplacementEffects$ ETBCreat
> SVar:ETBCreat:Event$ Moved | ValidCard$ Card.IsRemembered | Destination$ Battlefield | ReplaceWith$ DBPutP1P1 | ReplacementResult$ Updated | Description$ That creature enters the battlefield with X additional +1/+1 counters on it, where X is its mana value minus 4.
> SVar:DBPutP1P1:DB$ PutCounter | Defined$ ReplacedCard | CounterType$ P1P1 | ETB$ True | CounterNum$ Y
> ```
> 
> the only thing missing is to calculate Y, but for this i'm not sure if the value is calculated when the Trigger resolves or when the Creature enters

SVar:X:Remembered$CardManaCost/Minus.4 instead of triggeredCard made it work

File: forge-gui/res/cardsfolder/upcoming/ingenious_leonin.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Name:Ingenious Leonin
+ManaCost:4 W
+Types:Creature Cat Soldier
+PT:4/4
+A:AB$ PutCounter | Cost$ 3 W | ValidTgts$ Creature.attacking+Other+YouCtrl | TgtPrompt$ Select another target attacking creature you control | CounterType$ P1P1 | CounterNum$ 1 | SubAbility$ DBPump | SpellDescription$ Put a +1/+1 counter on another target attacking creature you control. If that creature is a Cat, it gains first strike until end of turn.
+SVar:DBPump:DB$ Pump | Defined$ Targeted | ConditionDefined$ Targeted | ConditionPresent$ Cat | KW$ First Strike
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/ingenious_leonin.txt:6]: Pretty sure this line will need some StackDesc help

File: forge-gui/res/cardsfolder/upcoming/magnanimous_magistrate.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Name:Magnanimous Magistrate
+ManaCost:5 W
+Types:Creature Human Advisor
+PT:3/4
+K:etbCounter:REPR:5:no condition:CARDNAME enters the battlefield with 5 reprieve counters on it.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/magnanimous_magistrate.txt:5]: I feel like this should work with just `K:etbCounter:REPR:5`

File: forge-gui/res/cardsfolder/upcoming/runadi_behemoth_caller.txt [hunk 3]
```diff
@@ -0,0 +1,15 @@
+Name:Runadi, Behemoth Caller
+ManaCost:2 G
+Types:Legendary Creature Cat Shaman
+PT:1/3
+T:Mode$ SpellCast | ValidCard$ Creature.cmcGE5 | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigEffect | TriggerDescription$ Whenever you cast a creature spell with mana value 5 or greater, that creature enters the battlefield with X additional +1/+1 counters on it, where X is its mana value minus 4.
+SVar:TrigEffect:DB$ Effect | RememberObjects$ TriggeredCard | ReplacementEffects$ ETBCreat | ExileOnMoved$ Stack
+SVar:ETBCreat:Event$ Moved | ValidCard$ Card.IsRemembered | Destination$ Battlefield | ReplaceWith$ DBPutP1P1 | ReplacementResult$ Updated
+SVar:DBPutP1P1:DB$ PutCounter | Defined$ ReplacedCard | CounterType$ P1P1 | ETB$ True | CounterNum$ X | SubAbility$ DBExile
+SVar:DBExile:DB$ ChangeZone | Defined$ Self | Origin$ Command | Destination$ Exile
+S:Mode$ Continuous | Affected$ Creature.YouCtrl+counters_GE3_P1P1 | AddKeyword$ Haste | Description$ Creature you control with three or more +1/+1 counters on them have haste.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/runadi_behemoth_caller.txt:10]: bad Desc (also Oracle)

File: forge-gui/res/cardsfolder/upcoming/mild_mannered_librarian.txt [hunk 4]
```diff
@@ -0,0 +1,9 @@
+Name:Mild-Mannered Librarian
+ManaCost:G
+Types:Creature Human
+PT:1/1
+A:AB$ Animate | Cost$ 3 G | GameActivationLimit$ 1 | Types$ Werewolf | Duration$ Permanent | SubAbility$ DBCounter | SpellDescription$ CARDNAME becomes a werewolf. Put two +1/+1 counters on it and you draw a card. Activate only once.
```
[the remark this answers is not in the document]
Comment (Hanmac) [reply forge-gui/res/cardsfolder/upcoming/mild_mannered_librarian.txt:5]: it also overwrites creature types (ala not in addition to its other types)

File: forge-gui/res/cardsfolder/upcoming/magnanimous_magistrate.txt [hunk 5]
```diff
@@ -2,7 +2,7 @@ Name:Magnanimous Magistrate
 ManaCost:5 W
 Types:Creature Human Advisor
 PT:3/4
-K:etbCounter:REPR:5:no condition:CARDNAME enters the battlefield with 5 reprieve counters on it.
+K:etbCounter:REPR:5:CARDNAME enters the battlefield with 5 reprieve counters on it.
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/magnanimous_magistrate.txt:5]: the Description should be auto generated and should not be needed

File: forge-gui/res/cardsfolder/upcoming/zask_skittering_swarmlord.txt [hunk 6]
```diff
@@ -0,0 +1,12 @@
+Name:Zask, Skittering Swarmlord
+ManaCost:3 G G
+Types:Legendary Creature Insect
+PT:5/5
+S:Mode$ Continuous | Affected$ Land.YouOwn,Insect.YouOwn | MayPlay$ True | AffectedZone$ Graveyard | Description$ You may play lands and cast Insect spells from your graveyard.
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Insect.YouCtrl | Execute$ TrigChange | TriggerDescription$ Whenever another Insect you control dies, put it on the bottom of its owner's library, then mill two cards. (Put the top two cards of your library into your graveyard.)
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/zask_skittering_swarmlord.txt:6]: Shouldn't trigger from own death

**Review by tool4ever — APPROVED** [review]
Runadi probably needs release notes later for confirmation?
Otherwise seems good for now

**Pull request merged**